owncloudApp/src/main/java/com/owncloud/android/ui/preview/PreviewImageFragment.kt

```diff
@@ -285,23 +285,18 @@ class PreviewImageFragment : FileFragment() {
     }
 
     override fun onFileMetadataChanged() {
-        val storageManager = mContainerActivity.storageManager
-        if (storageManager != null) {
-            file = storageManager.getFileByPath(file.remotePath)
-        }
+        file = mContainerActivity.storageManager?.getFileByPath(file.remotePath)
```

Comment:
In this case, do we want to set file to null when storage manager is null? Previously file was only updated when storage manager was not null.

Reply:
It doesn't mean that before it was good. Show nothing, previous file.... no clue. All doesn't feel good.

Anywhay the `?` was pointless. I removed it

Reply:
Btw the construct with `mContainerActivity` is questionable, but out of this topic

Reply:
> It doesn't mean that before it was good.

I agree. I was just wondering because it had different behavior.